banzai/tests/test_saving_qc.py

```diff
@@ -11,6 +11,7 @@ def test_format_qc_results_basic_info():
     assert results['instrument'] == image.instrument
     assert results['dayobs'] == image.epoch
     assert results['@timestamp'] == image.dateobs
+    assert results['obstype'].upper() == image.obstype
```

Comment:
Should obstype be upper case since it came from the FITS header? Or should we use full words here?

Reply:
I put obstype into elasticsearch as lowercase to try and stick with elasticsearch naming conventions, but actually since this is the field value and not the field name then it's probably more correct to just keep whatever comes out of the header as-is. I've removed the case changes.